app/languages/dictionaries/ml-utf8.csv

```diff
@@ -0,0 +1,831 @@
+aa
```

Comment:
A list of 800 words isn't going to be usable. Such short lists are suitable for full-sized QWERTY-like keyboards, where you can type all words even without suggestions from the keyboard. On the contrary, dictionaries for T9 keyboards must contain thousands or even millions of words to produce meaningful results. It's because the keyboard must "guess" what words match the digits.